spec/requests/sample_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+describe SampleController, type: :request do
+  describe "GET /sample" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.